docs/pages/server-access/guides/openssh-teleport.mdx

```diff
@@ -0,0 +1,338 @@
+---
```

Comment:
I assumed that's one-to-one port so not reviewing this part.

Reply:
No major changes - I did validate this and the other PAM doc. Some SEO and editing changes coming in.